exercises/concept/take-a-number/.docs/instructions.md

```diff
@@ -15,12 +15,15 @@ Note that each time you run this code, the PID may be different.
 
 ## 2. Report the machine state
 
-Modify the machine so that it can receive `{:report_state, sender_pid}` messages. It should send its current state (the last given out ticket number) to `sender_pid` and then wait for more messages.
+Modify the machine so that the newly spawned process is ready to receive messages (start a _receive loop_) with an initial state of `0`. It should be able to receive `{:report_state, sender_pid}` messages. As a response to those messages, it should send its current state (the last given out ticket number) to `sender_pid` and then wait for more messages.
```

Comment:
I hope that the phrase "receive loop" will remind people of the paragraph with the same title from the introductions

Reply:
Have you considered adding some example code in the receive loop section? I think code will make it a lot more memorable. Something like:

```elixir
def loop(ping_count) do
  receive do
    {:ping, sender_pid} ->
      send(sender_pid, :pong)
      loop(ping_count + 1)
  end
end
```

It's pretty close to the solution though.

Reply:
Yes, many times, but it is clearly against the recommendation:

> Code examples should only be used to introduce new syntax (students should not need to search the web for examples of syntax). In other cases provide descriptions or links instead of code.

Defining a recursive function is not new syntax...